main.rb

```diff
@@ -93,29 +95,27 @@ def login()
 
 def check_status(access_token)
   count = 1
+  puts "Selected Plan ID: #{$plan_id}".blue
   uri = URI.parse("https://testinium.io/Testinium.RestApi/api/plans/#{$plan_id}/checkIsRunning")
 
-  while is_count_less_than_max_api_retry(count) do
+  while is_count_less_than_max_api_retry(count)
     check_timeout()
-    req = Net::HTTP::Get.new(uri.request_uri,
-                             { 'Content-Type' => 'application/json', 'Authorization' => "Bearer #{access_token}", 'current-company-id' => "#{$company_id}" })
-    res = Net::HTTP.start(uri.hostname, uri.port, use_ssl: true) do |http|
-      http.request(req)
-    end
+    req = Net::HTTP::Get.new(uri.request_uri, { 'Content-Type' => 'application/json', 'Authorization' => "Bearer #{access_token}", 'current-company-id' => $company_id })
+    res = Net::HTTP.start(uri.hostname, uri.port, use_ssl: true) { |http| http.request(req) }
 
-    if (res.kind_of? Net::HTTPSuccess)
+    if res.is_a?(Net::HTTPSuccess)
       if get_parsed_response(res.body)[:running]
-        puts('Plan is still running...')
+        puts "Plan is still running...".yellow
         sleep($time_period)
       else
-        puts('Plan is not running...')
+        puts "Plan is not running.".green
```

Comment:
```suggestion
        puts "Plan is not running.".blue
```
IMO. or .red should be used. Since this expression is not a positive meaning. And we are returning if logic not allow to go to down side of code.